packages/cli/src/zosfiles/-strings-/en.ts

```diff
@@ -348,6 +348,23 @@ export default {
                     EX2: `Download the members of the data set "ibmuser.cntl" in text mode to the directory "jcl/"`
                 }
             },
+            ALL_MEMBERS_MATCHING: {
+                SUMMARY: "Download all members from a pds",
```

Comment:
Thoughts on capitalizing "pds" here so it matches the other references to PDS in this file?